### ISSUE TITLE
Tab is missing

### ISSUE THREAD

**Issue by bahamadigital** [kmgalanakis/Inceptive-Multiple-Categories-for-K2#8]
Hi,

Seems that the tab is missing from the K2 item editor. The K2 item list still has the Additional Categories column and all previously added extra categories seem to be still there. But I cannot add extra categories to items anymore because the tab is gone.

Joomla 3.8.1 and K2 2.8.0, Inceptive MCfK2 version is downloaded from GitHub on October 19th, 2017 (today) so it should be the latest version. 

Thank you!

**Comment by kmgalanakis** [comment]
Hello.

Could you please add some pictures that can better describe your situation, because to be honest, I cannot understand your issue.

Thanks

**Comment by bahamadigital** [comment]
Hello, 

Thank you for your reply. 

The problem was solved by moving the website to another server and reinstalling the plugin and setting up the extra categories again. So the problem is solved for now. 

But to illustrate what was wrong in the first place I added two screenshots to explain what I wrote in my previous message. 

In the first screenshot, there is the K2 items list. The Additional categories column (marked with number 1) was there as it should be. No problem there.  
![x-k2list](https://user-images.githubusercontent.com/25563974/35411625-1bca99d6-0222-11e8-947d-6a4682e1a427.jpg)

But if I clicked open any of the K2 items, the "Inceptive Multiple Categories for K2" tab (marked with number 2 in the screenshot below) **was not there**. It had disappeared and I couldn't get it back no matter what I tried. This screenshot shows the tab because it is from my current website which works as it should. I add the screenshot to clarify my original message.
![x-k2item](https://user-images.githubusercontent.com/25563974/35411624-1ba7345a-0222-11e8-854f-949d7240a05f.jpg)

Anyway, problem solved. Thank you for this great plugin!
